### PR TITLE
graph inserted in card

### PR DESCRIPTION
**Pull request by karthikgeek1**

### CODE REVIEW

File: src/Components/Cards/Cards.jsx [hunk 1]
```diff
@@ -1,19 +1,25 @@
 import React from 'react';
+import Graph3 from '../Graphs/Graph3';
 
 import './Cards.css'
 
-const Cards = () => {
+const Cards = (props) => {
```
Comment (Souvikdeb2612) [inline src/Components/Cards/Cards.jsx]: destructure props here

File: src/Components/Cards/Cards.jsx [hunk 2]
```diff
@@ -1,19 +1,25 @@
 import React from 'react';
+import Graph3 from '../Graphs/Graph3';
```
Comment (Souvikdeb2612) [inline src/Components/Cards/Cards.jsx]: 1 line space before internal imports

File: src/Components/Graphs/Graph1.jsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, createContext } from "react";
 import axios from "axios";
 import { Column } from "@ant-design/plots";
 
 import "./Graph1.css";
+export const Graphdata1 = createContext()
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph1.jsx]: create separate context folder

File: src/Components/Graphs/Graph2.jsx [hunk 4]
```diff
@@ -1,8 +1,9 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, createContext } from "react";
 import axios from "axios";
 import { Line } from "@ant-design/plots";
 
 import "./Graph2.css";
+export const Graphdata2 = createContext()
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph2.jsx]: same as above

File: src/Components/Graphs/Graph3.jsx [hunk 5]
```diff
@@ -1,9 +1,9 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, createContext } from "react";
 import axios from "axios";
 import { Area } from "@ant-design/plots";
 
 import "./Graph3.css";
-
+export const Graphdata3 = createContext()
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph3.jsx]: same as above

File: src/Components/Layout/StockLayout.jsx [hunk 6]
```diff
@@ -1,24 +1,34 @@
 import { Layout } from "antd";
 
-import React from "react";
+import React, {useState, useEffect, useContext} from "react";
 import Cards from "../Cards/Cards";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:5]: 1 line space before importing internal components

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Cards/Cards.jsx (hunk 1 above)
Comment (karthikgeek1) [reply src/Components/Cards/Cards.jsx]: done sir

File: src/Components/Cards/Cards.jsx (hunk 2 above)
Comment (karthikgeek1) [reply src/Components/Cards/Cards.jsx]: done sir

File: src/Components/Graphs/Graph1.jsx (hunk 3 above)
Comment (karthikgeek1) [reply src/Components/Graphs/Graph1.jsx]: done sir

File: src/Components/Graphs/Graph3.jsx (hunk 5 above)
Comment (karthikgeek1) [reply src/Components/Graphs/Graph3.jsx]: done sir

File: src/Components/Layout/StockLayout.jsx (hunk 6 above)
Comment (karthikgeek1) [reply src/Components/Layout/StockLayout.jsx:5]: done sir

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**